resources/js/zipassets.js

```diff
@@ -0,0 +1,27 @@
+$(function(){
+	$(".elementselect").each(function(){
+		var $assetwrapper = $(this);
+		var linkurl = "/actions/zipAssets/download?filename=assets.zip";
```

Comment:
If I use `Craft.postActionRequest` it returns the zip file in the `response`, which is probably not what we want.  Can we compromise and use `Craft.getActionUrl` and link to that instead?